projects/super_res/model/autoreg_diffusion.py

```diff
@@ -1312,6 +1311,7 @@ def sample(self):
         self.ema.ema_model.eval()
 
         cmap = mpl.colormaps['viridis']
+        sm = smap(None, cmap)
```

Comment:
I'm a little confused here. What if `config.data_config.logscale = False`? Does this mean that we don't want to plot in logscale anymore?

Reply:
that's just the default value in the function. it takes the value from config file where it's set to be true

Reply:
Sorry, I still don't get it. What happens if the `config.logscale` is False? I think the data will not be plotted in logscale but we always want to plot logscale, right?

Reply:
not just for plotting but as an input to the model itself